src/connectors/__test__/collectionService.test.ts

```diff
@@ -105,6 +105,59 @@ test('findAndCountCollectionsByUser', async () => {
   expect(records3[1].id).toBe(id1)
 })
 
+test('findArticleInCollection with the id', async () => {
+  const { id: collectionId } = await collectionService.createCollection({
+    authorId: '1',
+    title: 'test',
+  })
+  await collectionService.addArticles(collectionId, ['1', '2', '3', '4'])
+
+  const [articles1, totalCount] =
+    await collectionService.findArticleInCollection(collectionId, '3', {
+      take: 2,
+      reversed: false,
+    })
+
+  expect(articles1.length).toBe(2)
+  expect(totalCount).toBe(4)
+  expect(articles1[0].articleId).toBe('3')
```

Comment:
add `expect(articles3[1].articleId).toBe('4')` to make sure it's in right order.